### PR TITLE
Adding Recommend/Reuse Feature

### PR DESCRIPTION
**Pull request by KrisCovert**
Adding the recommend feature. This feature allows the user to automatically re-enter past items into their new list.

### CODE REVIEW

**Comment by KrisCovert** [comment]
Currently the feature adds all of the past items into the new list. Future implementation should allow the user to pick which items they want to add. Let me know what you guys think of my work!

**Comment by justincrawford-wsu** [comment]
Overall looks really really good. I'm just a tad confused on the MyList.js file and what that does. I think it is the list that is actually displayed on the AddDelete page right?

**Comment by lustyun** [comment]
Great job on the feature addition! I was also thinking the same thing as Justin. Is it recommending random items in a list from the history list? where is the recommended list shown? is it at the add/delete page?

**Comment by justincrawford-wsu** [comment]
@lustyun I do know that the recommended list is displayed on the AddDelete page

**Comment by Adrian-Tarnowski** [comment]
It looks very good, I think it myList.js just like a button that upon pressing it adds item from recommended list.

File: App/components/Items.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import { StyleSheet, View, Image, TouchableOpacity, Text } from "react-native";
-
-export default function Items({ pressHandler, item }) {
```
Comment (JacobSchnur) [inline App/components/Items.js:4]: Why did you have to remove pressHandler here? I thought that when we are trying to press a button (in this case add items) we need to pass in pressHandler.

File: App/components/MyListItems.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import { StyleSheet, View, Image, TouchableOpacity, Text } from "react-native";
+
+export default function MyListItems({ item }) {
```
Comment (JacobSchnur) [inline App/components/MyListItems.js]: Is this the function you use to add the items to your list?

Comment (JacobSchnur) [inline App/components/MyListItems.js]: Is this the function you use to add the items to your list?

File: App/components/recommend.js [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+        submitHandler(
+          groceryItems.name,
+          groceryItems.price,
+          "to be determined"
+        );
+      }
+    });
+  };
+
+  // const recycleWhenPressed2 = () => {
```
Comment (JacobSchnur) [inline App/components/recommend.js]: We can probably delete this if it is not being used.

**Review by JacobSchnur — COMMENTED** [review]
It looked good to me. The only thing I did not understand fully what it did was the MyListItems.js. Other than that can't wait to pull this and see how it looks. Good work.

**Comment by KrisCovert** [comment]
My bad guys. I didn't realize I pushed some things that weren't supposed to be pushed (MyListItems.js was not supposed to be pushed, it does nothing). I will resolve the conflicts right now.

File: App/components/Items.js (hunk 1 above)
Comment (KrisCovert) [reply App/components/Items.js:4]: My I was unaware that I deleted this functionality, it is definitely important! I will edit my file to include it!

**Comment by JacobSchnur** [comment]
Okay

> My bad guys. I didn't realize I pushed some things that weren't supposed to be pushed (MyListItems.js was not supposed to be pushed, it does nothing). I will resolve the conflicts right now.

Looks good now! Good job Kris!

**Comment by JacobSchnur** [comment]
My apologies did not mean to close the request I re-opened it

**Comment by KrisCovert** [comment]
> Overall looks really really good. I'm just a tad confused on the MyList.js file and what that does. I think it is the list that is actually displayed on the AddDelete page right?

The MyListItems.js file was mistakenly pushed. The recommend list is displayed in a pop up window on the AddDelete page you are correct.

**Comment by KrisCovert** [comment]
> Great job on the feature addition! I was also thinking the same thing as Justin. Is it recommending random items in a list from the history list? where is the recommended list shown? is it at the add/delete page?

The recommend component is taking the all the items from the MyList database, so nothing is rondom here! The recommend list is shown on the AddDelete page as a pop up window.

File: App/components/recommend.js (hunk 3 above)
Comment (KrisCovert) [reply App/components/recommend.js]: Correct! I will delete it right now.

**Pull request merged**